controllers/k8ssandra/stargate.go

```diff
@@ -192,3 +198,104 @@ func (r *K8ssandraClusterReconciler) removeStargateStatus(kc *api.K8ssandraClust
 		}
 	}
 }
+
+func (r *K8ssandraClusterReconciler) reconcileStargateConfigMap(
+	ctx context.Context,
+	remoteClient client.Client,
+	kc *api.K8ssandraCluster,
+	desiredConfig map[string]interface{},
+	userConfigMapContent,
+	namespace string,
+	logger logr.Logger,
+) result.ReconcileResult {
+	logger.Info("Reconciling Stargate Cassandra yaml configMap on namespace : " + namespace)
+	filteredCassandraConfig := filterYamlConfig(desiredConfig["cassandra-yaml"].(map[string]interface{}))
```

Comment:
These values always need to be the same as what's configured in the Cassandra datacenter, otherwise Stargate won't be able to communicate with the nodes. Even when we'll expose the Cassandra settings in a structured manner, encryption options should not be exposed.

Reply:
I've added a comment explaining this on the encryption stores structure of the CRD.